Docs/HowFuzzilliWorks.md

```diff
@@ -428,7 +428,7 @@ As can be imagined, minimization is very expensive, frequently requiring over a
 It is possible to tune the minimizer to remove code less aggressively through the `--minimizationLimit=N` CLI flag. With that, it is possible to force the minimizer to keep minimized programs above a given number of instructions. This can help retain some additional code fragments which might facilitate future mutations. This can also speed up minimization a bit since less instructions need to be removed. However, setting this value too high will likely result in the same kinds of problems that the minimizer attempts to solve in the first place.
 
 ### Corpus
-Implementation: [Corpus.swift](https://github.com/googleprojectzero/fuzzilli/blob/master/Sources/Fuzzilli/Core/Corpus.swift)
+Implementation: [Corpus/](https://github.com/googleprojectzero/fuzzilli/tree/master/Sources/Fuzzilli/Corpus) subdirectory
```

Comment:
Thanks! I think the link should be `https://github.com/googleprojectzero/fuzzilli/tree/main/Sources/Fuzzilli/Corpus` though (branch = `main` instead of `master`)?